crypto/mem.c

```diff
@@ -305,6 +297,24 @@ int CRYPTO_mem_debug_pop(void)
     return -1;
 }
 
+void CRYPTO_mem_debug_malloc(void *addr, size_t num, int flag,
```

Comment:
Should these functions be surrounded by `#ifdef` deprecation guards?

Reply:
look up a few dozen lines.

Reply:
and not that @levitte approved this :)

Reply:
Oops, typo.  *Note* that Richard approved this.

Reply:
> look up a few dozen lines.

oh yes, I completely missed it

https://github.com/openssl/openssl/pull/10629/files#diff-0ad35796f3b4040e66893ac4cb4b6b28R274-R276




> _Note_ that Richard approved this.

Sometimes even @levitte can overlook things, that's why I was asking for confirmation ;)